de.fu_berlin.inf.dpp.server/src/de/fu_berlin/inf/dpp/server/editor/Editor.java

```diff
@@ -0,0 +1,79 @@
+package de.fu_berlin.inf.dpp.server.editor;
+
+import de.fu_berlin.inf.dpp.activities.SPath;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'de.fu_berlin.inf.dpp.activities.SPath'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=146&issues=AWY_D4ujvOM6zLh9KWhV&open=AWY_D4ujvOM6zLh9KWhV)

[ISSUE_KEY]: AWY_D4ujvOM6zLh9KWhV